types/src/v2/params.rs

```diff
@@ -182,7 +186,13 @@ impl<'a> RpcParamsSequence<'a> {
 
 				Some(Ok(value))
 			}
-			Err(_) => {
+			Err(e) => {
+				log::error!(
```

Comment:
this is super useful but I think we could include it in `InvalidParams(String)` too

Reply:
Fair, but that's a different PR I think.

Reply:
indeed

Reply:
https://github.com/paritytech/jsonrpsee/issues/460